libraries/java11/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Pull base image.
+RUN apt-get -y update && \
+    apt-get install --no-install-recommends -y \
+        software-properties-common \
+        build-essential \
+        ca-certificates \
+        libssl-dev \
+        apt-transport-https \
+        wget \
+        default-jdk \
+        openjdk-8-jdk \
```

Comment:
The rest of the file paths say `java11` so my guess is that this shouldn't be jdk-8 here? Is the default-jdk java11?

Reply:
good point, that will be updated. we want ojdk 11.